File: MyMovieDB/app/src/main/java/isel/pdm/demos/mymoviedb/MovieDto.java
package isel.pdm.demos.mymoviedb;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Class whose instances represent movie information obtained from the remote API.
 */
public class MovieDto {

    private boolean adult;

    public boolean getAdult() {
        return adult;
    }

    @JsonProperty("title")
    private String title;

    public String isTitle() {
        return title;
    }

    private long id;

    public long getId() {
        return id;
    }
}

File: MyMovieDB/app/src/main/kotlin/isel/pdm/demos/mymoviedb/models/MovieDto.kt
package isel.pdm.demos.mymoviedb.models

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * Class whose instances represent movie information obtained from the remote API.
 */
data class MovieDto(val adult: Boolean, val title: String, val id: Long) {
    private constructor() : this(false, "", 0)
}
